go/interpreter/lfvm/super_instructions.go

```diff
@@ -85,10 +85,8 @@ func opSwap2_Swap1(c *context) {
 func opDup2_Mstore(c *context) error {
 	var value = c.stack.pop()
 	var addr = c.stack.peek()
```

Comment:
since you are modifying the functions, is this address or offset ?

Reply:
offset